src/main/java/api/ApiRequestBuilder.java

```diff
@@ -1,42 +1,93 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
```

Comment:
Author is missing in copyright in all classes

Reply:
solved